.github/workflows/end-to-end-test-portals.yml

```diff
@@ -36,19 +36,33 @@ jobs:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 0
+      - name: Derive appropriate SHAs for base and head for `nx affected` commands
+        uses: nrwl/nx-set-shas@v4
+
       - uses: ./.github/actions/pnpm-setup-action
+      - name: Save affected projects to environment variable
+        run: |
+          pnpm nx show projects --affected
+          echo "AFFECTED_PROJECTS=$(pnpm nx show projects --affected)" >> "$GITHUB_ENV"
```

Comment:
Code fails when affected projects contains more than one project. I believe this is because pnpm will then output an array which cannot be passed to `GITHUB_ENV`

Reply:
Rather than passing the array of affected projects I should just pass a true/false boolean that can be used directly in the `if` parameters of the following steps.